julia-mode.el

```diff
@@ -243,7 +243,7 @@
    '("if" "else" "elseif" "while" "for" "begin" "end" "quote"
      "try" "catch" "return" "local" "function" "macro" "ccall"
      "finally" "break" "continue" "global" "where"
-     "module" "using" "import" "export" "const" "let" "do" "in"
```

Comment:
The commit message makes it clear why `in` was removed, and I agree, but would you mind adding a `julia--should-font-lock` test for `in` in `for ... in ...` expressions? And, possibly `for`, which we don't test for either.